### ISSUE TITLE
Intentionally die until spawning in the middle

### ISSUE THREAD

**Issue by ghost** [ermiyaeskandary/Slither.io-bot#166]
- [ ] Make bot suicide till it spawns in the middle (check if you spawned inside a certain square/circle using x and y co-ords)
- [x] Check for bigger food instead of clumps of food (Find out the size of the food relative to your head diameter)

~~I also don't know how to add labels (Github noob)~~ - You can't add labels if you aren't a contributor ~ ErmiyaEskandary

**Comment by chancity** [comment]
It's already does that when picking food.

**Comment by CJBoey** [comment]
Spawning at middle doesn't necessary make the bot performs any better. Also the bot performs badly at place where there are a lot of snakes around it, it mostly just spinning in circle and afraid to get those dead snake (when the bot is big enough >5k)

**Comment by ghost** [comment]
@CJBoey its for manual use

**Comment by tjorim** [comment]
I don't think that spawning untill you are in the center would be very useful.

**Comment by ghost** [comment]
@tjorim it can be so you can get mass faster then move out

**Comment by kakol20** [comment]
While it's true that food appear more in the middle however, it leaves you more vulnerable to dying and will more likely be targeted by the circle of death.

**Comment by ghost** [comment]
Yes but it's easier to get kills so you become big and then leave the mid

**Comment by ChadSki** [comment]
> Find out the size of the food relative to your head diameter

The physical size of food seems to have more to do with zoom artifacts -- when you zoom out, food stays the same size on screen and appears larger.

**Comment by Drflash55** [comment]
@ChadSki Like you said about how the food appears larger when zoomed out, there's not much you can do about that. You would need to have the game refresh the size of the food so that it's proportionally displayed correctly.

**Comment by vikasTmz** [comment]
Isn't the game tracking the performance from an IP? I've noticed the better I perform the more chances of spawning in the center. Seems like a strategy for them to let bigger/better players come in the center, so they will be encouraged to play further. And they also might be tracking the food cluster size, so if you've had a long game , they'll spawn you next to big food cluster in any server.
